src/PlaneWaveBasis.jl

```diff
@@ -325,6 +325,12 @@ The wave vectors `G` in reduced (integer) coordinates for a cubic basis set
 of given sizes.
 """
 function G_vectors(fft_size::Union{Tuple,AbstractVector})
+    start = .- cld.(fft_size .- 1, 2)
+    stop  = fld.(fft_size .- 1, 2)
+    axes  = [[collect(0:stop[i]); collect(start[i]:-1)] for i in 1:3]
+    [Vec3{Int}(i, j, k) for i in axes[1], j in axes[2], k in axes[3]]
+end
+function G_vectors_generator(fft_size::Union{Tuple,AbstractVector})
     start = .- cld.(fft_size .- 1, 2)
     stop  = fld.(fft_size .- 1, 2)
     axes = [[collect(0:stop[i]); collect(start[i]:-1)] for i in 1:3]
```

Comment:
have one `collect` the other?

Reply:
and a comment saying what this is for

Reply:
With the collect it's 100-fold slower than with the code duplication.

Reply:
wow really? It must hit the captured variables in closure thing, or something similar. OK then just add a comment saying that?